src/Shopsys/ShopBundle/DataFixtures/Demo/CountryDataFixture.php

```diff
@@ -44,6 +44,7 @@ public function load(ObjectManager $manager): void
         $countryData->names = [
             'cs' => 'Česká republika',
             'en' => 'Czech republic',
+            'de' => 'Tschechische Republik',
```

Comment:
I would expect Germany would be actually added as a country for a German demoshop...